src/app/organizer/ItemTable.tsx

```diff
@@ -567,6 +191,21 @@ function ItemTable({
     useRowSelect
   ) as TableInstance<DimItem> & UseRowSelectInstanceProps<DimItem>;
 
+  const hiddenColumns: string[] = useMemo(
+    () =>
+      columns
+        .flatMap((c) => {
+          if (c.id && !enabledColumns.includes(c.id)) {
+            const subColumnIds = c.columns?.map((sub) => sub.id) || [];
+            return [c.id, ...subColumnIds];
+          }
+        })
+        .filter(isDefined),
+    [columns, enabledColumns]
+  );
+
+  useEffect(() => setHiddenColumns(hiddenColumns), [setHiddenColumns, hiddenColumns]);
```

Comment:
Is this really meant to be used from `useEffect`? From [the react-table documentation](https://github.com/tannerlinsley/react-table/blob/master/docs/api/useTable.md#instance-properties) it looks like `show` should still work, but also that the hidden columns functions are meant to be passed directly into a column selection widget like ours.